meshroom/core/graph.py

```diff
@@ -1094,7 +1094,7 @@ def dfsMaxEdgeLength(self, startNodes=None):
         """
         nodesStack = []
         edgesScore = defaultdict(lambda: 0)
-        visitor = Visitor(reverse=False, dependenciesOnly=False)
+        visitor = Visitor(reverse=False, dependenciesOnly=True)
```

Comment:
`dfsMaxEdgeLength` is not something used for nodes alignment in the UI? In which case, dependenciesOnly should be False.

Reply:
Maybe not the case anymore.
Propably better to just expose dependenciesOnly as a parameter of `dfsMaxEdgeLength`.